src/components/ebay-area-chart/area-chart.stories.js

```diff
@@ -0,0 +1,88 @@
+import { addRenderBodies } from '../../../.storybook/utils';
+import Readme from './README.md';
+import Component from './index.marko';
+
+function getPositiveTrend(days) {
+    const data = [];
+    const length = days !== undefined ? days : 30;
+    for (let i = 0; i < length; i++) {
+        const d = new Date(`Feb ${i + 1}, 2022 00:00:00 UTC`);
+
+        let v = i === 0 ? 0 : data[i - 1].y;
+        v = parseFloat((v + Math.random() * 5000 - Math.random() * 2500).toFixed(2));
+        if (v < 0) {
+            v = 0;
+        }
+        data.push({
+            x: d.getTime(),
+            y: v,
+            label: `$${v}`,
+        });
+    }
+
+    return {
+        data,
+    };
+}
+
+const multipleLineSeries = (num, days) => {
```

Comment:
I removed this with the last commit, this was just one of my random chart data generator helper functions.  I've since just made a hard coded data object.